docs/install.rst

```diff
@@ -437,7 +437,7 @@ installing ``astropy`` with pip::
 You can alternatively install the `sphinx-astropy
 <https://github.com/astropy/sphinx-astropy>`_ package with pip::
 
-    python -m pip install sphinx-astropy
+    python -m  sphinx-astropy
```

Comment:
```suggestion
    python -m sphinx-astropy
```

Reply:
good point, I may have been low on coffee if I didn't see this. Let me update my suggestion.

Reply:
Shouldn't have been. I think this happened in the rebase. I'll edit.

Reply:
Done.